bool open
int64[] snack_ids
---
